docs/install/initial_setup.rst

```diff
@@ -1,18 +1,61 @@
 .. _setup_initial:
 
-
 Initial Setup
 =============
 
-To do the initial setup of your Kolibri facility after the installation, follow these steps.
+Setting up your Kolibri server will differ depending on your learning environment. If you plan to use Kolibri at home or for supplemental learning outside any facility, use **Quick start**; if you are installing Kolibri to be used by many learners at a formal facilities like schools, or non-formal like libraries or community centers, use **Advanced setup**.
 
 .. note::
-  You need to do the initial setup only once, the first time you start Kolibri after the installation. You can change all the settings later, *except the type and the name of the facility*.
+  You need to do the initial setup only once, the first time you start Kolibri after the installation. You can change all the settings later.
+
+Quick Start
+-----------
+
+To do the initial setup of Kolibri for use at home, follow these steps.
+
+
+#. **Select the default language** for Kolibri.
+
+  	.. figure:: /img/select-language.png
+	   :alt:  
+
+	   Select the default Kolibri language.
+
+
+	.. warning::
+	  The default language configured for content in your browser preferences might override the language you choose in this step. To ensure that Kolibri displays in the desired language, make sure to configure it as default in the browsers of all the devices that will be used to view Kolibri content.
```

Comment:
is this still true? I know it was at some point but I thought we changed it

Reply:
I'm not 100% sure that it will not happen, and since so many factors are in play (OS display language, preferred language for content display in browser, device default language Kolibri setting, user set Kolibri language), I'd prefer if we keep it for now.